Add callout on the cap for sequencer.l1-confs and verifier.l1-confs (#866)
ethereum-optimism/docs: pages/builders/node-operators/configuration/consensus-config.mdx

  <Tabs.Tab>`OP_NODE_SEQUENCER_L1_CONFS=4`</Tabs.Tab>
</Tabs>

<Callout type="warning">

> **Important:** The maximum value for `sequencer.l1-confs` cannot exceed the sequencer drift, which is currently set to 30 minutes (1800 seconds or 150 blocks). Configuring a value higher than the allowed limit will prevent the sequencer from producing blocks within the sequence window.

Reviewer: ```suggestion
The maximum value for `sequencer.l1-confs` cannot exceed the sequencer drift, currently set to 30 minutes (1800 seconds or 150 blocks). Setting a value higher than this limit will prevent the sequencer from producing blocks within the sequence window.
```

  <Tabs.Tab>`OP_NODE_VERIFIER_L1_CONFS=0`</Tabs.Tab>
</Tabs>

<Callout type="info">

> **Suggested:** While `verifier.l1-confs` does not have a strict limit, it is recommended to keep this value within 12-13 minutes (realistically 10-20 blocks) to ensure optimal performance. Configuring this parameter beyond the recommended range may impact the verifier's ability to process data efficiently.

Reviewer: ```suggestion
While `verifier.l1-confs` has no strict limit, it's recommended to keep this value within 12-13 minutes (typically 10-20 blocks) for optimal performance. Exceeding this range may impact the verifier's data processing efficiency.
```
